components/progressbar/stories/template.js

```diff
@@ -31,6 +33,8 @@ export const Template = ({
 			style=${styleMap({
 				"width": customWidth,
 				...customStyles,
+				"--mod-progressbar-track-color": trackFill ? trackFill : undefined,
+				"--mod-progressbar-fill-color": progressBarFill ? progressBarFill : undefined,
```

Comment:
We can shorten the assignment here:

```suggestion
				"--mod-progressbar-track-color": trackFill,
				"--mod-progressbar-fill-color": progressBarFill,
```

Reply:
absolutely, good call- https://github.com/adobe/spectrum-css/pull/2929/commits/bafdfd090d965a4c1fd5e77c6887c96da7e0d2b3